kafka-inmemory/resources/openshift-template.yaml

```diff
@@ -3,6 +3,11 @@ kind: Template
 metadata:
   name: barnabas
 parameters:
+- description: Number of Zookeper cluster nodes which will be deployed
```

Comment:
No harm in mentioning that an odd number is required to achieve a quorum in the event of network partition.

Reply:
Fixed.